GVRf/Framework/jni/engine/renderer/renderer.cpp

```diff
@@ -49,24 +49,147 @@ void Renderer::renderCamera(std::shared_ptr<Scene> scene,
+                            //If visibility flag was set by an earlier occlusion query,
+                            //turn visibility on for the object
+                            if(visible)
+                                render_data_vector.push_back(render_data);
+#if _GVRF_USE_GLES3_
+                            if (render_data->material() != 0)
+                            {
+                                if(scene->get_occlusion_culling())
+                                {
+									//If a previous query is active, do not issue a new query.
+									//This avoids overloading the GPU with too many queries
+									//Queries may span multiple frames
+
+                                	bool is_query_issued = (*it)->is_query_issued();
+                                	if(!is_query_issued)
+                                	{
+										//Setup basic bounding box and material
+										std::shared_ptr<RenderData> bounding_box_render_data(new RenderData());
+										std::shared_ptr<Mesh> bounding_box_mesh = render_data->mesh()->getBoundingBox();
+										bounding_box_render_data->set_mesh(bounding_box_mesh);
+
+                                        GLuint *query = (*it)->get_occlusion_array();
+
+                                        glDepthFunc(GL_LEQUAL);
+                                        glEnable(GL_DEPTH_TEST);
+                                        glColorMask(GL_FALSE, GL_FALSE, GL_FALSE, GL_FALSE);
+
+                                        //Issue the query only with a bounding box
+                                        glBeginQuery(GL_ANY_SAMPLES_PASSED, query[0]);
+                                        shader_manager->getBoundingBoxShader()->render(mvp_matrix_tmp, bounding_box_render_data);
+                                        glEndQuery(GL_ANY_SAMPLES_PASSED);
+                                        (*it)->set_query_issued(true);
+
+                                        glColorMask(GL_TRUE, GL_TRUE, GL_TRUE, GL_TRUE);
+
+                                        //Delete the generated bounding box mesh
+                                        bounding_box_mesh->cleanUp();
+                                    }
+                                }
+                            }
+#endif
+                        }
+                        else
+                        {
+                            (*it)->set_in_frustum(false);
+                        }
+                        delete bounding_box_info;
+                    }
+                    else
+                    {
+                        //No occlusion or frustum tests enabled
+                        render_data_vector.push_back(render_data);
+                    }
                 }
             }
         }
         std::sort(render_data_vector.begin(), render_data_vector.end(),
                 compareRenderData);
 
-        glm::mat4 view_matrix = camera->getViewMatrix();
-        glm::mat4 projection_matrix = camera->getProjectionMatrix();
```

Comment:
you ended up renaming this to proj_matrix up above and didn't change it accordingly on lines 217 and 233.  the build broke.